app/views/declaration/summary/parties_section.scala.html

```diff
@@ -14,101 +14,57 @@
  * limitations under the License.
  *@
 
-@import forms.common.Address
-@import models.declaration.Parties
+@import models.requests.JourneyRequest
 @import models.viewmodels.HtmlTableRow
 
-@(parties: Option[Parties])(implicit messages: Messages)
+@(declarationData: ExportsDeclaration)(implicit messages: Messages, journeyRequest: JourneyRequest[AnyContent])
```

Comment:
Similar

Reply:
For every section we will stay with declarationData as argument. We will use implicit one on the `summaryPage`